cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -0,0 +1,115 @@
+
+{# ==========================================================================
+
+   post_preview.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render an article when given:
+
+   settings:
+
+   settings.post_title:         A string with the title of the post.
+   settings.post_content:       A string with content of post preview/post summary.
+   settings.author:             An array with authors of the post.
+   settings.published_date:     A datetime object with published date.
+
+   settings.image_url:          A string with the url of the thumbnail image.
+   settings.image_alt:          A string with the alt text of the thumbnail image.
+
+   settings.post_category:      An array with the categories for the post. Up to 2.
+   settings.post_tags:          An array with the post tags for the post.
+
+   settings.event_location:     A string with the event location.
+   settings.event_date:         A datetime object with the time of the event.
+
+   settings.comments_close_date: A datetime object marking the deadline of the comment period.
+   settings.external_link:      A string with the description text of an external link.
+   settings.external_link_url:  A string with the url of an external link.
+
+   path:                        A string with the path for the homepage of the pagetype
+                                `/blog/`, `/newsroom/`
+
+   ========================================================================== #}
+
+{% macro render(settings, path) %}
+
+{% import 'macros/share.html' as share with context %}
+{% import 'category-slug.html' as category_slug %}
+{% import 'macros/time.html' as time %}
+
+<article class="o-post-preview">
+    <div class="meta-header">
+        <span class="date meta-header_right">
+            Published {{ time.render(settings.published_date, {'date':true}) }}
+        </span>
+        {% if settings.post_category.0 %}
+            {% set use_post_category = true %}
```

Comment:
The spec says the max number of categories to display is two. I can do a loop, but would it still be worth it for that? LMK.